src/main/java/core/basesyntax/service/filesoperation/WriteDataToFileImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.filesoperation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteDataToFileImpl implements WriteDataToFile {
+    private static final String REPORT_HEADER = "fruit,quantity";
+    private static final String ERROR_WRITE_MESSAGE = "Can't write data to file ";
+
+    @Override
+    public void writeData(String outputFile) {
```

Comment:
The same as in reader. This method should only write to the file, what to write is to receive from outside.